rag/nlp/search.py

```
@@ -638,6 +643,12 @@ async def retrieval(
                     vector_similarity_weight,
                     rank_feature=rank_feature,
                 )
+            elif settings.DOC_ENGINE_GAUSSDB:
+                # GaussDB Path B returns final SQL-side scores from the adapter.
+                sim = [sres.field[id].get("_score", 0.0) for id in sres.ids]
```

Ruff (0.16.0)

[error] Variable `id` is shadowing a Python builtin

(A001)